libs/utils/src/getRouteTitle.ts

```diff
@@ -0,0 +1,17 @@
+import { i18nKeys } from '@form-exercise/i18n';
+import { routesArray, IRoutesArray } from '@form-exercise/utils';
+
+interface IGetRouteTitle {
+  pathname: string;
+}
+
+export const getRouteTitle = ({ pathname }: IGetRouteTitle): string => {
+  const routes: IRoutesArray[] = routesArray;
+  const routeIndex = routes.findIndex((route) => route.path === pathname);
```

Comment:
move to a new funtion, `getRoute` that returns the content of the route (routeName, routePath):
```ts
export  const getRoute = (pathname: string) => routes.find((route) => route.path === pathname);

```